src/main/java/com/example/hop_oasis/model/Rating.java

```diff
@@ -0,0 +1,27 @@
+package com.example.hop_oasis.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+public class Rating {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long itemId;
+    @Enumerated(EnumType.STRING)
+    private ItemType itemType;
+    private double rating;
```

Comment:
double means that you'll have a default 0.0 value for missing values. 
There is a difference in 0.0 and null values for the rating.
0.0 means that rating is very bad
null means that rating is absent